src/preconditions/ClientPermissions.ts

```diff
@@ -55,6 +55,8 @@ export class CorePrecondition extends AllFlowsPrecondition {
 	): AllFlowsPrecondition.AsyncResult {
 		const required = context.permissions ?? new PermissionsBitField();
 
+		if (!interaction.channel) return this.ok(); // When using user-installable apps channel will be always null
```

Comment:
Mmm, I am not ok with defaulting to true when user-apps commands are ran. We should either be checking interaction.appPermissions or figure out a better way to handle user apps 😅